lib/rules/max-lines.js

```diff
@@ -139,7 +139,8 @@ module.exports = {
                 if (lines.length > max) {
                     context.report({
                         loc: { line: 1, column: 0 },
-                        message: "File must be at most " + max + " lines long."
+                        message: "File must be at most " + max + " lines" +
+                        " long. It's " + lines.length + " lines long."
```

Comment:
I think you could actually leave everything in the same line for increased readability: 
```javascript
message: "File must be at most " + max + " lines long. It's " + lines.length + " lines long."
```